tools/cldr-code/src/main/java/org/unicode/cldr/util/Emoji.java

```diff
@@ -312,14 +306,6 @@ public static String getMajorCategory(String emoji) {
         return majorCat;
     }
 
-    public static Set<String> getMajorCategories() {
```

Comment:
Why delete these? Remember that the unicodetools uses the CLDR APIs, so you can't assume that public APIs are only used internal to CLDR. We don't want to accidentally break unicodetools

Reply:
I'm glad you pointed that out! Is there nothing in the cldr project to indicate which APIs are needed for unicodetools? Just now I did a search in unicodetools for getMajorCategories and the result was nothing found; here's a screenshot:
![image](https://github.com/unicode-org/cldr/assets/24991506/071900ce-db10-4cd1-9811-6cf4422e1ffa)

Reply:
To answer the question, "Why delete these?": dead code is detrimental in many ways, especially for maintainability. The ability to distinguish dead code from live code is essential. Do we have no way to do that within the cldr project?

Reply:
Not without verifying that unicodetools still works. That is the main project that uses some of the CLDR api.